src/ui/utils/timeline.ts

```diff
@@ -173,50 +86,14 @@ export function getSecondsFromFormattedTime(formatted: string) {
   return minutes * 60 * 1000 + seconds * 1000 + milliseconds;
 }
 
-export function isSameTimeStampedPointRange(
-  range1: TimeStampedPointRange,
-  range2: TimeStampedPointRange
-) {
-  const sameBegin =
-    range1.begin.point === range2.begin.point && range1.begin.time === range2.begin.time;
-  const sameEnd = range1.end.point === range2.end.point && range1.end.time === range2.end.time;
-
-  return sameBegin && sameEnd;
-}
-
-export function isInFocusSpan(time: number, focusWindow: FocusWindow) {
+export function isInFocusSpan(time: number, focusWindow: TimeStampedPointRange) {
   return time >= focusWindow.begin.time && time <= focusWindow.end.time;
 }
 
-export function isPointInRegions(regions: TimeStampedPointRange[], point: string) {
-  return regions.find(
-    r => BigInt(point) >= BigInt(r.begin.point) && BigInt(point) <= BigInt(r.end.point)
-  );
-}
-
-export function isTimeInRegions(time: number, regions?: TimeStampedPointRange[]): boolean {
-  return !!regions?.some(region => time >= region.begin.time && time <= region.end.time);
-}
-
-export function rangeForFocusWindow(focusWindow: FocusWindow): TimeStampedPointRange {
+export function rangeForFocusWindow(focusWindow: TimeStampedPointRange): TimeStampedPointRange {
   return { begin: focusWindow.begin || { time: 0, point: "0" }, end: focusWindow.end };
```

Comment:
This method looks funky:
```js
export function rangeForFocusWindow(focusWindow: TimeStampedPointRange): TimeStampedPointRange {
  return { begin: focusWindow.begin || { time: 0, point: "0" }, end: focusWindow.end };
}
```

Either the types are wrong (I don't think they are) or it doing...nothing.

I think it was unnecessary before but maybe _looked_ necessary because it accepted a `FocusWindow` and returned a `TimeStampedPointRange` (even though those types were the same).

Kill it?